doc/quickstart/osm_quickstart.rst

```diff
@@ -106,6 +96,84 @@ PostgreSQL for map data, but does not work directly with OSM files.
 * Homepage: https://wiki.openstreetmap.org/wiki/Osm2pgsql
 * Tutorial: https://learnosm.org/en/osm-data/osm2pgsql/
 
+
+Working with OSM data in QGIS
+================================
+
+OSGeoLive comes with OSM Data for the city where the global `FOSS4G conference <https://foss4g.org>`_ takes place.
```

Comment:
OSM data

Reply:
Oh yes! QMS. Sure, thanks, I will add the images  :)